.pre-commit-config.yaml

```
@@ -27,11 +27,27 @@ repos:
         name: Update BSP README with dependencies section
         entry: .github/ci/update_readme_dependencies.py
         language: python
-        files: 'bsp\/.*\/include\/.*\.h|bsp\/.*\/idf_component\.yml|bsp\/.*\/README\.md' # All idf_component.yml, README.md and .h files in bsp directory
+        files: 'bsp/.*/include/bsp/.*\.h|bsp/.*/idf_component\.yml|bsp/.*/README\.md' # All idf_component.yml, README.md and .h files in bsp directory
         additional_dependencies:
           - idf_component_manager==1.4.2
           - py-markdown-table
 
+  - repo: local
+    hooks:
+      - id: update-supported-boards
+        name: Update Supported Boards Table
+        entry: python .github/ci/update_supported_boards.py
+        language: system
```

Nitpick: a little cleaner would be
```suggestion
        entry: .github/ci/update_supported_boards.py
        language: python
```